src/masternodes/rpc_accounts.cpp

```diff
@@ -1117,6 +1125,10 @@ UniValue listaccounthistory(const JSONRPCRequest& request) {
             return true;
         }
 
+        if(txn != std::numeric_limits<uint32_t>::max() && txn != key.txn) {
+            return true;
+        }
+
```

Comment:
hmm. @chanakasameera two things `filtration` and `pagination` within the same RPC. need to be careful. 
https://github.com/DeFiCh/jellyfish/issues/1036 requirement is to address slice pagination.
This way it's acting like a filter that we don't want and hardly someone would use.